### PR TITLE
Update my-app.html

### PR DESCRIPTION
**Pull request by nika-begiashvili**
use let instead of var

### CODE REVIEW

File: src/my-app.html [hunk 1]
```diff
@@ -157,7 +157,7 @@
 
       _pageChanged(page) {
         // Load page import on demand. Show 404 page if fails
-        var resolvedPageUrl = this.resolveUrl('my-' + page + '.html');
+        let resolvedPageUrl = this.resolveUrl('my-' + page + '.html');
```
Comment (nidelson) [inline src/my-app.html]: If the reference (resolvedPageUrl) will not be changed, I would use it as constant.
```js
const resolvedPageUrl = this.resolveUrl('my-' + page + '.html');
```

Comment (abdonrd) [reply src/my-app.html]: Yep! 👍

Comment (nika-begiashvili) [reply src/my-app.html]: yes that's more like it

Comment (nidelson) [reply src/my-app.html]: @nika-begiashvili Update the PR for @abdonrd to make the merge for us.

**Comment by abdonrd** [comment]
Thanks @nika-begiashvili & @nidelson! 👏

**Pull request merged**